contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
```

Comment:
I believe we'll need to support an instance in one namespace being used by an app in another - so they can share data. And I think this is why the "instanceRef" property isn't just a name but a full ObjectReference. Otherwise, yes, I agree if they share the same namespace then all we'd need is the name.